osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -20,6 +24,8 @@ public class PlaySongSelect : SongSelect
         private OsuScreen player;
         private readonly ModSelectOverlay modSelect;
         private readonly BeatmapDetailArea beatmapDetails;
+        private IEnumerable<Mod> originalMods;
+        private bool controlPressed;

```

Comment:
If anything, store a Func<bool> that you create in `load()`. Don't want a local instance of `UserInputManager` just for this